### DISCUSSION TITLE
Should we pre-print our manuscript? If so, where?

### DISCUSSION THREAD

**Discussion by wavingtowaves** [SORTEE-Github-Hackathon/manuscript#174] [Polls]
Hi everyone, @DrMattG brought up a great point in #173 that I'd like the group input on.

The journal we plan to submit to, Nature E&E supports authors pre-printing their manuscripts. So does this (interesting article)[https://onlinelibrary.wiley.com/doi/full/10.1002/ece3.8106] in the journal _Ecology and Evolution_ . Are we OK with pre-printing, and if so, where do you think?  If you choose 'other' please let us know where you prefer as a comment.

**Comment by colebrookson** [comment]
Big proponent of pre-printing, definitely worth doing so if we can!

**Comment by adbinley** [comment]
ditto on the pre-print, I think it's a great idea. I don't actually have strong opinions on where, but I voted for bioRxiv because it is the one I come across most frequently

**Comment by wavingtowaves** [comment]
Thanks for the input all, and also the journal we are planning to submit to is supportive of pre-prints too 👍🏻
